src/actions/auth.js

```diff
@@ -0,0 +1,69 @@
+import firebase from 'firebase';
+
+const firebaseApp = {
+  apiKey: 'AIzaSyDwnsJcOAy2cP2lyTFdWcanOMTf2DRZpIs',
+  authDomain: 'local-cb336.firebaseapp.com',
+  databaseURL: 'https://local-cb336.firebaseio.com',
+  storageBucket: 'local-cb336.appspot.com',
+  messagingSenderId: '514582937003'
+};
+
+firebase.initializeApp(firebaseApp);
+
+const provider = new firebase.auth.GoogleAuthProvider();
+
+export const startListeningToAuth = () => {
+  return (dispatch, getState) => {
+    firebase.auth().onAuthStateChanged(user => {
+      if (user) {
```

Comment:
Can you log the value of `user` here and let me know what it spits back? If it's complaining about `displayName` not existing, we want to see if this object doesn't have a `displayName` property for some reason. If it doesn't, we'll want to use a different value like an email address or anything else that identifies who is logged in.